### PR TITLE
LPS-161789 Enhance the request with the info-item title

### PR DESCRIPTION
**Pull request by BarbaraCabrera**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/ui/BreadcrumbUtil.java [hunk 1]
```diff
@@ -334,6 +335,15 @@ private static void _addLayoutBreadcrumbEntries(
 				themeDisplay.getLocale(), "control-panel");
 		}
 
+		if (layout.isTypeAssetDisplay()) {
+			String name = (String)httpServletRequest.getAttribute(
```
Comment (lfbesada) [inline portal-kernel/src/com/liferay/portal/kernel/servlet/taglib/ui/BreadcrumbUtil.java:338]: rename variable to infoItemName

**Review by lfbesada — CHANGES_REQUESTED** [review]

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:stop

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:relevant

**Comment by BarbaraCabrera** [comment]
ci:test:relevant

**Comment by BarbaraCabrera** [comment]
ci:stop

**Comment by BarbaraCabrera** [comment]
ci:relevant

**Comment by BarbaraCabrera** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:stop

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:stop

**Pull request closed without merging**